src/Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -6,8 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.Interactive" Version="$(MicrosoftDotNetInteractiveVersion)" />
-    <PackageReference Include="Microsoft.DotNet.Interactive.Formatting" Version="$(MicrosoftDotNetInteractiveVersion)" />
```

Comment:
does `Microsoft.DotNet.Interactive.Formatting` reference is not needed any more here?